.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,132 @@
+name: Docker Build
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build:
+    name: Build Docker Image
+    runs-on: ubuntu-22.04
+    permissions:
+      packages: write
+    strategy:
```

Comment:
Some idea:
Use max-parallel to build some images in parallel ? 
Use fail-fast: false to build all images even if some of them fail ?

Reply:
> Use max-parallel to build some images in parallel ?

That's already the case, they are all built in parallel.

> Use fail-fast: false to build all images even if some of them fail ?

I think if one build fails there is not much use for the other images.  I would rather go in the opposite direction and prevent the publishing of all images if a single one cannot be built, but not in this PR.